tests/pipelines/animatediff/test_animatediff.py

```diff
@@ -401,6 +402,64 @@ def test_free_init_with_schedulers(self):
                 "Enabling of FreeInit should lead to results different from the default pipeline results",
             )
 
+    def test_free_noise_blocks(self):
```

Comment:
I've added two tests for FreeNoise based on what I think are the most important parts. LMK if anything else is needed